File: src/main/java/com/sfh/pokeRogueBot/model/browser/gamejson/StarterDataEntry.java
package com.sfh.pokeRogueBot.model.browser.gamejson;

import com.google.gson.annotations.SerializedName;
import lombok.Data;

@Data
public class StarterDataEntry {

    @SerializedName("moveset")
    private Object moveset;

    @SerializedName("eggMoves")
    private int eggMoves;

    @SerializedName("candyCount")
    private int candyCount;

    @SerializedName("friendship")
    private int friendship;

    @SerializedName("abilityAttr")
    private int abilityAttr;

    @SerializedName("passiveAttr")
    private int passiveAttr;

    @SerializedName("valueReduction")
    private int valueReduction;

    @SerializedName("classicWinCount")
    private int classicWinCount;
}

File: src/main/kotlin/com/sfh/pokeRogueBot/model/browser/gamejson/StarterDataEntry.kt
package com.sfh.pokeRogueBot.model.browser.gamejson

import com.google.gson.annotations.SerializedName

data class StarterDataEntry(
    @SerializedName("moveset")
    val moveset: Any?,

    @SerializedName("eggMoves")
    val eggMoves: Int,

    @SerializedName("candyCount")
    val candyCount: Int,

    @SerializedName("friendship")
    val friendship: Int,

    @SerializedName("abilityAttr")
    val abilityAttr: Int,

    @SerializedName("passiveAttr")
    val passiveAttr: Int,

    @SerializedName("valueReduction")
    val valueReduction: Int,

    @SerializedName("classicWinCount")
    val classicWinCount: Int
)